rust/ruby-rbs/build.rs

```diff
@@ -195,6 +208,91 @@ fn write_visit_trait(file: &mut File, config: &Config) -> Result<(), Box<dyn std
             "pub fn visit_{}_node<V: Visit + ?Sized>(visitor: &mut V, node: &{}Node) {{",
             method_name, node_variant_name
         )?;
+
```

Comment:
We need to remove the `allow(unused_variables)` above.

Reply:
What do you think of prefixing the parameters with an underscore instead of turning off the lint? Something like this:

```rust
if !has_visitable_fields {
  // If there's nothing to visit in this
  // node, write the empty method with
  // the parameters using an underscore
  // and skip to the next iteration
  writeln!(
    file,
    "pub fn visit_{}_node<V: Visit + ?Sized>(_visitor: &mut V, _node: &{}Node) {{}}",
    method_name, node_variant_name
  )?;

  continue;
}
```